our-work/software/provident.yml

```diff
@@ -0,0 +1,15 @@
+title: Provident
+description: A web app to support the PROVIDENT research study looking to prevent drug-related deaths in neighborhoods across Rhode Island. The dashboard showcases a predictive analytics model that identifies neighborhoods where future overdose deaths are most likely to occur. We are also tracking interventions from organizations and what effects they will have on the overdose data. The results from this study will improve public health decision-making and inform resource allocation to communities that should be prioritized for evidence-based prevention, treatment, recovery, and overdose rescue services.
+languages:
+  - javascript
+  - vue
```

Comment:
All these yaml go through validation provided by the [corresponding schema](https://github.com/brown-ccv/ccv-website-content/blob/main/schemas/software.yml). You'll need to adjust some of your tags, I think `vue` should be `vuejs` and you can add new ones to the schema (i.e `typescript` and `react-native`). There are some instructions in the main README if you want to validate via `yamale` locally. If not, check the logs of the `software` validation

Reply:
I changed `vue` --> `vuejs`. I also added `react-native` to the schema